test/contact_bloc_test.mocks.dart

```diff
@@ -0,0 +1,44 @@
+// Mocks generated by Mockito 5.1.0 from annotations
+// in contacts_data/test/contact_bloc_test.dart.
+// Do not manually edit this file.
+
+import 'dart:async' as _i4;
+
+import 'package:contacts_data/model/contact.dart' as _i5;
+import 'package:contacts_data/service/service_contact.dart' as _i3;
+import 'package:http/http.dart' as _i2;
+import 'package:mockito/mockito.dart' as _i1;
+
+// ignore_for_file: type=lint
```

Comment:
you to leave the files a a little more clean, you could remove the commented code and the "as _iX" added in the build generated files. It will look better

Reply:
that anyway are autogenerated files, but understood I take it out